index.js

```diff
@@ -1,96 +1,23 @@
 require('coffee-script/register')
-
-// Use bluebird for performance
-global.Promise = require('bluebird')
-
-// Merge sharify data
```

Comment:
This file is cleaned up quite a bit now that we don't have to do weird things to merge code from desktop and mobile.